resolver/resolver.cc

```diff
@@ -2262,8 +2282,64 @@ class ResolveSignaturesWalk {
         }
     }
 
-    void fillInInfoFromSig(core::MutableContext ctx, core::SymbolRef method, core::LocOffsets exprLoc, ParsedSig sig,
-                           bool isOverloaded, const ast::MethodDef &mdef) {
+    static void handleAbstractOrInterfaceMethods(core::Context ctx, ast::MethodDef &mdef) {
```

Comment:
What is an interface method? afaik there are only abstract methods.

Reply:
Poorly named method; I can rename it 😓  `handleAbstractMethods` captures it since methods defined on an interface are de-facto abstract.